libs/components/progress-indicator/src/lib/modules/progress-indicator/progress-indicator-item/progress-indicator-item.component.ts

```diff
@@ -40,14 +40,15 @@ export class SkyProgressIndicatorItemComponent implements OnInit {
   /**
    * A help key that identifies the global help content to display. When specified, a [help inline](https://developer.blackbaud.com/skyux/components/help-inline) button is
    * placed beside the progress indicator item label. Clicking the button invokes global help as configured by the application.
+   * This property only applies when `title` is also specified.
    */
   @Input()
   public helpKey: string | undefined;
 
   /**
    * The content of the help popover. When specified along with `labelText`, a [help inline](https://developer.blackbaud.com/skyux/components/help-inline)
```

Comment:
```suggestion
   * The content of the help popover. When specified along with `title`, a [help inline](https://developer.blackbaud.com/skyux/components/help-inline)
```
I assume this should be `title` since that's what the last sentence we're adding calls out. But if `labelText` is correct, I assume that means the last sentence should be changed to match.